applitools/package.json

```diff
@@ -0,0 +1,23 @@
+{
+  "name": "applitools",
+  "version": "1.0.0",
+  "description": "This is a sample for applitools",
```

Comment:
It should be "Applitools".

Reply:
this is a name of test, company name => keep this